service/controller/resources/vaultcrt/desired_test.go

```diff
@@ -58,29 +60,29 @@ func Test_Resource_VaultCrt_GetDesiredState(t *testing.T) {
 		// Test 1 is the same as 0 but with a different custom object.
 		{
 			Obj: &v1alpha1.CertConfig{
+				ObjectMeta: apismetav1.ObjectMeta{
+					Labels: map[string]string{
+						"cert-operator.giantswarm.io/version": project.Version(),
+					},
+				},
 				Spec: v1alpha1.CertConfigSpec{
 					Cert: v1alpha1.CertConfigSpecCert{
 						ClusterID:        "al9qy",
 						ClusterComponent: "worker",
 					},
-					VersionBundle: v1alpha1.CertConfigSpecVersionBundle{
-						Version: "0.2.0",
-					},
 				},
 			},
 			ExpectedSecret: &apiv1.Secret{
 				ObjectMeta: apismetav1.ObjectMeta{
 					Name: "al9qy-worker",
 					Annotations: map[string]string{
-						ConfigHashAnnotation:           "4bf7b5296ba01161f182de54b243e1400ae6660e",
-						UpdateTimestampAnnotation:      (time.Time{}).Format(UpdateTimestampLayout),
-						VersionBundleVersionAnnotation: "0.2.0",
+						ConfigHashAnnotation:      "4bf7b5296ba01161f182de54b243e1400ae6660e",
+						UpdateTimestampAnnotation: (time.Time{}).Format(UpdateTimestampLayout),
 					},
 					Labels: map[string]string{
-						"clusterID":                 "al9qy",
```

Comment:
`ClusterID` and `ClusterComponent` have been dropped from `certs` -- is there a reason to include them on future Secrets?